forge-gui/res/cardsfolder/upcoming/mirror_of_life_trapping.txt

```diff
@@ -0,0 +1,10 @@
+Name:Mirror of Life Trapping
+ManaCost:4
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.wasCast | Execute$ TrigExile | TriggerZones$ Battlefield | TriggerDescription$ Whenever a creature enters the battlefield, if it was cast, exile it, then return all other permanent cards exiled with CARDNAME to the battlefield under their owners’ control.
```

Comment:
change `’` to `'` here and in line 10

Reply:
not sure if i've succeeded in fixing this. lmk.